README.md

```diff
@@ -4490,6 +4598,10 @@ kubectl run nginx --image=nginx --restart=Never --port 80 --expose
 <summary>Describe in detail what the following command does <code>kubectl create deployment kubernetes-httpd --image=httpd</code></summary><br><b>
 </b></details>
 
+<details>
```

Comment:
You already added this

Reply:
I didn't get you, the 4601 `<details>` should not be there? 

below is the sytax
```
<details>
<summary>Why to create kind deployment, if pods can be launched with replicaset ? </summary><br><b>
</b></details>
```

Reply:
I'm just saying that these questions (Kubernetes) are already added by you in a previous PR. I think you need to rebase your PR to only add the Ansible questions.